bin/webwork3

```diff
@@ -10,5 +10,14 @@ use Mojo::File qw(curfile);
 use lib curfile->dirname->sibling('lib')->to_string;
 use Mojolicious::Commands;
 
+# check if the config file has been created
+my $webwork_dir = curfile->dirname->dirname("..");
+
+warn "The file $webwork_dir/conf/webwork3.yml does not exist.  Perhaps you haven't copied webwork3.dist.yml to this file."
```

Comment:
```suggestion
warn "The file $webwork_dir/conf/webwork3.yml does not exist.  Perhaps you haven't copied webwork3.yml.dist to this file."
```
The .dist extension is after the .yml extension on the actual file.  Perhaps we should switch that though so that linters and editors properly recognize the file type?